src/psyclone/tests/psyir/nodes/reference_test.py

```diff
@@ -40,7 +40,8 @@
 
 from __future__ import absolute_import
 import pytest
-from psyclone.psyir.nodes import Reference, Array, Assignment, Literal
```

Comment:
This file has pycodestyle errors.

Reply:
Fixed these. pylint has one warning about too many statements. Should I disable or leave?

Reply:
Disable please.

Reply:
Disabled.